### PR TITLE
Added Wifi Connected Device Script and Readme.md file

### PR DESCRIPTION
**Pull request by anubhab1710**
Fulfilled issue no: #133 
@atarax665 Please note that the "no new line at EOF" is just an indication and not an error. There's no other problem with the code. Thanks for the cooperation. 
### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### Description:
This PR has the folder Wifi Connected Device containing the Python Script and descriptive Readme.md file for implementing the find users connected to Wifi script (as per the assigned issue)

#### Added
1. Main script code (in .py format)
2. Readme.md file for description of the script

### Snapshot:
Wifi

### CODE REVIEW

**Comment by atarax665** [comment]
> @atarax665 Please note that the "no new line at EOF" is just an indication and not an error. There's no other problem with the code. Thanks for the cooperation. 

@anubhab1710 I guess I didn't said that no new line at the EOF is an error. While reviewing code we don't just look for syntax and logical errors in the code. We need to make sure that contributors follow the best coding practices while writing code and also the Readme file is structured according to the project's code base so that the project looks uniform.

**Comment by atarax665** [comment]
You could have made the suggested changes in the previous PR itself and should have squashed the commits later. Always make sure to pull a request from a new branch (except main).

File: Python/Wifi Connected Device/WifiDevices-Readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+On the console, type `import subprocess`
+## Running the Script:
+After opening the script in your Python IDE, execute the code so that you get the console output window. 
+The script will print the SSIDs and usernames of all the connected devices. The SSID is the name of wireless network consisting of unique 32 alphanumeric characters for each user.
+## Basic Terminologies:
+>`netsh`: is a command-line scripting utility that allows you to display or modify the network configuration of a computer that is currently running. <br>
+>`wlan`: command that allows you to access the Wi-Fi profiles. <br>
+>`network`: the unique name of wireless network. 
+## Output:
+<img src="Screenshot.PNG"><br>
```
Comment (atarax665) [inline Python/Wifi Connected Device/WifiDevices-Readme.md:18]: Change this to markdown syntax.

**Review by atarax665 — COMMENTED** [review]
Rename ```WifiDevices-Readme.md``` to ```README.md```.

**Comment by Tejas1510** [comment]
@anubhab1710, @atarax665 is just helping you to be a good developer, he has very good experience in this field and is just sharing his knowledge to make you better which is the main motto of open source
I hope you will try to learn as much from all of us which will be very useful to you in your development journey
Happy Coding

**Comment by anubhab1710** [comment]
> @anubhab1710, @atarax665 is just helping you to be a good developer, he has very good experience in this field and is just sharing his knowledge to make you better which is the main motto of open source
> I hope you will try to learn as much from all of us which will be very useful to you in your development journey
> Happy Coding

Thanks for showing the way @Tejas1510 .and @atarax665 . I didn't face this issue while making my previous PRs in this repo that's why I felt I was doing the right thing till now, but since I was made to make proper changes, now I understand how to tackle new issues in GitHub. Thanks again :)

**Comment by Tejas1510** [comment]
Great going @anubhab1710 
If you face any doubts we were always there was help

**Pull request closed without merging**